website/cue/reference/components/sources/internal_metrics.cue

```diff
@@ -1073,22 +1089,32 @@ components: sources: internal_metrics: {
 			required:    true
 			examples: ["http://localhost:8080/server-status?auto"]
 		}
+		_error: {
+			description: "The specifics of the error condition, such as system error code, etc."
+			required:    true
+			examples: ["File not found", "Connection dropped by remote"]
+		}
```

Comment:
Are there are concerns about downstream services handling tag values like this? After further thought, they do seem a little non-standard with what I'm familiar with. What would you think of leaving this tag out until someone asks for it?

Reply:
That was my concern as well, that and a potential for cardinality issues.